ansible/playbooks/Supporting_Scripts/Nagios_Ansible_Config_tool/templates/template.cfg

```diff
@@ -3,57 +3,42 @@ define host {
         notification_interval           30
         notification_period             24x7
 	notifications_enabled           0
 }
-define service{
-        use                             local-service
-	host_name                       ReplaceHostName
-	service_description             SSH
-	check_command			check_ssh
-        }
-
-define service{
-        use                             local-service
-        host_name                       ReplaceHostName
-        service_description             PING
-	check_command			check_ping!200.0,20%!500.0,60%
-        }
 
 define service{
         use                             generic-service        
 	host_name                       ReplaceHostName
         service_description             Current Load
         check_command                   check_by_ssh!/usr/local/nagios/libexec/check_load -w 15,10,5 -c 30,24,20
-        }
-
-define service{
-        use                             generic-service        
-        host_name                       ReplaceHostName
-        service_description             Total Processes
-        check_command                   check_by_ssh!/usr/local/nagios/libexec/check_procs -w 170 -c 200
-        }
+        check_interval                  30
+}
 
 define service{
         use                             generic-service        
         host_name                       ReplaceHostName
-        service_description             Zombie Processes
-        check_command                   check_by_ssh!/usr/local/nagios/libexec/check_procs -w 5 -c 10 -s Z
-        }
+        service_description             Disk Space Root Partition
+        check_command                   check_by_ssh!/usr/local/nagios/libexec/check_disk -w 20% -c 10% -p /
+        check_period                    once-a-day-at-8
+}
 
 define service{
-        use                             generic-service        
+        use                             generic-service
         host_name                       ReplaceHostName
-        service_description             Current Users
-        check_command                   check_by_ssh!/usr/local/nagios/libexec/check_users -w 5 -c 10
-        }
+        service_description             Check Jenkins Connection
+        check_command                   check_agent!ReplaceHostName
+        check_period                    once-a-day-at-8
+}
 
 define service{
-        use                             generic-service        
+        use                             generic-service
         host_name                       ReplaceHostName
-        service_description             Disk Space Root Partition
-        check_command                   check_by_ssh!/usr/local/nagios/libexec/check_disk -w 20% -c 10% -p /
-        }
+        service_description             Check Network Time System
+        check_command                   check_by_ssh!/usr/lib/nagios/plugins/check_timesync
+        check_period                    once-a-day-at-8
```

Comment:
Whether the time is synced should not depend on the workload of the system.

Reply:
What implies it's dependent on the workload of the system?

Reply:
I generally don't understand why we don't check everything regularly in short intervals, e.g. every 5 or 15 minutes, because that's how you do it. Somebody argued in another discussion that we should do some workload dependant checks once a day. Even if I don't agree, time synchronization isn't workload dependant.

Reply:
Ah I see, okay, yeah, I can put the check interval down